Makefile

```diff
@@ -1,10 +1,13 @@
 SHELL = bash
 
+# --no-print-directory avoids verbose logging when invoking targets that utilize sub-makes
+MAKE_OPTS ?= --no-print-directory
```

Comment:
This doesn't seem to work on my machine. I don't get different output when running make build. Is this for a particular make command?

Reply:
This was added to address verbose output from CI agents.  Indeed, my local version doesn't see any change with this added, either.